docs/docs/deploying-to-netlify.md

```diff
@@ -27,13 +27,17 @@ Netlify currently has built in support for [GitHub](https://github.com/), [GitLa
 
 Now, login to Netlify and you will see a `New site from git` button at the top right corner of the screen. Click on it and connect with the same git provider that you used to host your website and authorize Netlify to use your account. Choose your website repository and it will take you to deploy settings with the below options.
 
-- Branch to deploy: You can specify a branch to monitor. When you push to that particular branch, only then will Netlify build and deploy the site. The default is `master`.
+- Branch to deploy: You can specify a branch to monitor. When you push to that particular branch, only then will Netlify build and deploy the site. The default is `master`. The secondary default is `main`.
```

Comment:
Please revert that change as this isn't documented by Netlify and I assume this just happened to you because the `main` branch became the `default` branch by deleting `master` and thus Netlify takes the default one.